src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+.html {
+  --header-link-color: #00acdc;
+}
```

Comment:
It seems you are trying to style the html element, but the class name '.html' does not refer to the html element. If you want to style the html element, you should use 'html' without the dot.